jbehave-core/src/main/java/org/jbehave/core/embedder/PerformableTree.java

```diff
@@ -1464,22 +1469,6 @@ private List<PendingStep> pendingSteps() {
             return pending;
         }
 
-        private void generatePendingStepMethods(RunContext context, List<PendingStep> pendingSteps) {
-            if (!pendingSteps.isEmpty()) {
-                PendingStepMethodGenerator generator = new PendingStepMethodGenerator(context.configuration()
-                        .keywords());
-                List<String> methods = new ArrayList<>();
-                for (PendingStep pendingStep : pendingSteps) {
-                    if (!pendingStep.annotated()) {
-                        methods.add(generator.generateMethod(pendingStep));
-                    }
-                }
-                context.reporter().pendingMethods(methods);
-                if (context.configuration().pendingStepStrategy() instanceof FailingUponPendingStep) {
```

Comment:
do we still have ability to not fail on pending steps?

Reply:
+1, it seems `pendingStepStrategy()` is completely ignored now